nav2_costmap_2d/test/integration/costmap_tests_launch.py

```diff
@@ -29,8 +30,16 @@ def main(argv=sys.argv[1:]):
     launchFile = os.path.join(os.getenv('TEST_LAUNCH_DIR'), 'costmap_map_server.launch.py')
     testExecutable = os.getenv('TEST_EXECUTABLE')
 
+    lifecycle_manager = launch_ros.actions.Node(
+        package='nav2_lifecycle_manager',
+        node_executable='lifecycle_manager',
+        node_name='lifecycle_manager',
+        output='screen',
+        parameters=[{'node_names': ['map_server']}, {'autostart': True}])
```

Comment:
How is the map server being launched? Adding it to the list of node names tells the lifecycle_manager to bring it up, but the map_server process must be started (the lifecycle manager doesn't launch it). I assume there is another launch file that's part of this test?

Reply:
Yes, `costmap_map_server.launch.py` does the launching of the map server.

https://github.com/ros-planning/navigation2/blob/178c7c6ce4aa3ea58189302e1287d44f800543cd/nav2_costmap_2d/test/test_launch_files/costmap_map_server.launch.py#L23